contrib/apply-patches.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+for f in $@ ; do
```

Comment:
Quotes around `"$@"` in case someone ever tries building in a path with a space in it.

Reply:
fixed